pkg/tcpip/header/icmpv4.go

```diff
@@ -150,6 +156,12 @@ func (b ICMPv4) SetSequence(sequence uint16) {
 	binary.BigEndian.PutUint16(b[icmpv4SequenceOffset:], sequence)
 }
 
+// Set Time for Timestamp reply messages
+func (b ICMPv4) SetRxTimeStamp(t_rx int64,t_tx int64)  {
```

Comment:
Use camelCase for variables names, and note minor spacing issue in the signature.